src/meta/proto-conv/tests/it/user_stage.rs

```diff
@@ -91,6 +91,7 @@ fn test_user_stage_oss_v13() -> anyhow::Result<()> {
         copy_options: mt::CopyOptions {
             on_error: mt::OnErrorMode::SkipFileNum(666),
             size_limit: 1038,
+            split_size: 0,
```

Comment:
Add test_user_stage_s3_v16() test_user_stage_gcs_v16() and test_user_stage_oss_v16()

Reply:
done